pkg/autoscaler/autoscaler_test.go

```diff
@@ -331,22 +331,7 @@ func (r *mockReporter) ReportRequestedPodCount(v int64) error {
 }
 
 // ReportActualPodCount of a mockReporter does nothing and return nil for error.
-func (r *mockReporter) ReportActualPodCount(v int64) error {
-	return nil
-}
-
-// ReportNotReadyPodCount of a mockReporter does nothing and return nil for error.
-func (r *mockReporter) ReportNotReadyPodCount(v int64) error {
-	return nil
-}
-
-// ReportPendingPodCount of a mockReporter does nothing and return nil for error.
-func (r *mockReporter) ReportPendingPodCount(v int64) error {
-	return nil
-}
-
-// ReportTerminatingPodCount of a mockReporter does nothing and return nil for error.
-func (r *mockReporter) ReportTerminatingPodCount(v int64) error {
+func (r *mockReporter) ReportActualPodCount(ready, notReady, terminating, pending int64) error {
```

Comment:
the name should probably be `ReportPodCounts` since `ActualPodCount` refers to the ready pods in the metrics.

I wonder if we should change `actual_pods` to `ready_pods` [here](https://github.com/knative/serving/blob/7a7646862e94c11f6a20a886543e477e75b45b65/pkg/autoscaler/stats_reporter.go#L41-L44) for the metrics to match better with the rest of the metrics we report. wdyt @vagababov @markusthoemmes ?